packages/nimble-components/.storybook/main.js

```diff
@@ -17,7 +17,8 @@ module.exports = {
         '@storybook/addon-interactions'
     ],
     features: {
-        previewCsfV3: true
+        previewCsfV3: true,
+        previewMdx2: true
```

Comment:
Needed to switch to the [experimental Mdx2](https://storybook.js.org/docs/react/writing-docs/mdx#mdx-2) to support React 18 as a peerDependency (which was required due to the limited React peerDependency range of the accessibility plugin).
Both CSF3 and MDX2 are no longer experimental in the future Storybook 17